package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "quarkify",
-  "version": "1.20.2",
+  "version": "1.20.3",
```

Comment:
version check please

Reply:
Confirmed to 2.0.0. Waiting for approval :)